connector.go

```diff
@@ -389,6 +389,58 @@ func (c *Connector) GetObject(
 	return
 }
 
+func (c *Connector) PagingGetObject(
+	obj IBObject, ref string,
+	queryParams map[string]string, res *[]interface{}) (err error) {
+
```

Comment:
Hi @skalpin , Can you please say how you tested this code with these attributes? Was the code working for you?

Reply:
@hemanthKa677 I haven't tested this code. The implementation I am using, and have tested is using golang 1.18.